### DISCUSSION TITLE
Is raspberry pi support has been fixed?

### DISCUSSION THREAD

**Discussion by rlarudqls630** [coder/code-server#4775] [General]
Is raspberry pi support has been fixed?

**Comment by lg-nineoneninethreenine** [comment]
It supposedly is, but there is no way you can download it right now till the next release. Moreover, there are some bugs too, but I think it's just my RasPi well enough.

**Reply by jsjoeio** [reply]
^Yes! You'll have to try when the next release is out (4.0.2)

**Reply by rlarudqls630** [reply]
I am so excited for this.
hope new version releases soon!!

is there estimated time line for the release?

**Reply by jsjoeio** [reply]
You can follow along [here](https://github.com/coder/code-server/milestone/35). We're hoping to hit everything and release by Jan. 28 (but sometimes those dates get pushed back).

If you're comfortable living on the bleeding edge, you should be able to use the `beta` version of code-server after [this PR](https://github.com/coder/code-server/pull/4769) is merged. 

Otherwise wait for the stable release

**Reply by lg-nineoneninethreenine** [reply]
> you should be able to use the `beta` version of code-server after this PR is merged.

It's merged now, does that mean there's a beta ready for us?

**Reply by rlarudqls630** [reply]
nice, right on my birthday!! 
it is like my birthday gift :D

**Reply by lg-nineoneninethreenine** [reply]
Happy birthday! But a small problem - the package isn't up yet. But maybe you could try getting it by building code-server.

**Reply by lg-nineoneninethreenine** [reply]
I found the error: https://github.com/coder/code-server/runs/4903326846?check_suite_focus=true#step:14:13

The tag needs to be set to `beta`.

**Reply by jsjoeio** [reply]
It _should_ be up 🤔 You can try installing with `yarn global add code-server@beta`. see [here](https://www.npmjs.com/package/code-server/v/latest)

Nice find @im-coder-lg! I'm not sure why that's happening. I thought I set the tag but I guess not.

**Reply by lg-nineoneninethreenine** [reply]
Link fail, can you check that? Also, I think it's up since there are two new releases not tagged latest after 4.0.1, any ideas?

**Reply by jsjoeio** [reply]
Hmm it's working for me? https://www.npmjs.com/package/code-server/v/latest

![image](https://user-images.githubusercontent.com/3806031/151070922-a4d1bcb2-51ed-4496-9da0-7ad9b6a8ade3.png)

Though `beta` isn't ready. @code-asher found a bug. Once https://github.com/coder/code-server/issues/4790 is fixed, you'll be able to install `beta` and try it out.

> Also, I think it's up since there are two new releases not tagged latest after 4.0.1, any ideas?

I assume you're talking about GitHub Releases? These are dev/beta releases and specific to npm so they aren't tagged with `git`.

**Reply by lg-nineoneninethreenine** [reply]
No, I am talking about the latest beta releases. At the NPM registry, there is a valid beta tag and some PR tags. Are those beta/dev releases?

**Reply by lg-nineoneninethreenine** [reply]
Also, you need to update the links there from `cdr` to `coder` yet again.

**Reply by jsjoeio** [reply]
> At the NPM registry, there is a valid beta tag and some PR tags. Are those beta/dev releases?

Yep! That's right. 

> Also, you need to update the links there from cdr to coder yet again.

Oops. That's in the `package.json`. Nice catch!

**Reply by lg-nineoneninethreenine** [reply]
lol, I am becoming more observant. Can I fix it if a PR is not underway?

**Reply by jsjoeio** [reply]
Yes!
